packages/react/src/components/Pagination/PaginationRoot.tsx

```diff
@@ -14,6 +14,11 @@ export const PaginationContext = createContext<PaginationContextProps>({
 });
 
 export type PaginationRootProps = {
+  /**
+   * Sets the screen reader label for the Pagination area
+   * @default 'Sidenavigering'
+   */
+  'aria-label'?: string;
```

Comment:
Was this supposed to be required? ( according to your changeset 😅 )

Reply:
Oh, Tobias said this should not be required after all, but I forgot to remove this from changeset- well spotted! I'll update :)